experiment/bump_e2e_image.sh

```diff
@@ -51,9 +51,9 @@ popd
 
 echo "TAG = ${TAG}"
 
-$SED -i "s/\/kubekins-e2e:.*$/\/kubekins-e2e:${TAG}-master/" "${TREE}/images/kubeadm/Dockerfile"
-$SED -i "s/\/kubekins-e2e:v.*$/\/kubekins-e2e:${TAG}-master/" "${TREE}/experiment/generate_tests.py"
-$SED -i "s/\/kubekins-e2e:v.*-\(.*\)$/\/kubekins-e2e:${TAG}-\1/" "${TREE}/experiment/test_config.yaml"
+$SED -i "s/\\/kubekins-e2e:.*$/\\/kubekins-e2e:${TAG}-master/" "${TREE}/images/kubeadm/Dockerfile"
+$SED -i "s/\\/kubekins-e2e:v.*$/\\/kubekins-e2e:${TAG}-master/" "${TREE}/experiment/generate_tests.py"
+$SED -i "s/\\/kubekins-e2e:v.*-\\(.*\\)$/\\/kubekins-e2e:${TAG}-\\1/" "${TREE}/experiment/test_config.yaml"
```

Comment:
how did this work before?

Reply:
\ followed by an unescaped character is just \ and the unescaped character-- but it's not really safe.

Reply:
thanks, that makes sense.

Reply:
xref: https://github.com/kubernetes/test-infra/pull/8538/files#r199619471